libraries/AP_NavEKF2/AP_NavEKF2_core.cpp

```diff
@@ -1440,9 +1440,11 @@ void NavEKF2_core::MagOffsetConstrain(void)
 {
     if (frontend->_mag_learn_limit > 0) {
         float limit_ga = frontend->_mag_learn_limit * 0.001f;
+        // allow higher limit on Z due to more variability observed in vehicles
+        float limit_ga_z = limit_ga * 3;
```

Comment:
Should 3 be #defined, const, or constexpr??

Reply:
see my GNC channel comment